crawl-ref/source/dat/des/branches/hells.des

```diff
@@ -0,0 +1,395 @@
+ ..cc...cc.c..
+..Dc.cD2..DcD..
+.Dc2c...cc.2c..
+.c.c..D...c.cc.
+c.D..cDcDD.c.c.
+D.c.c....c..c.D
+c.2.Dc1}.2c.D2c
+.c..c....cD.c.c
+Dc....cD.c..c.D
+.cD.c...c..c.c.
+.Dc2.DccD...2c.
+..ccD..2..cDc..
+ ..D.cccDcDc..
+  ......cc...
+ENDMAP
+
+NAME:    nicolae_hells_fiendish_ambush
+TAGS:    transparent no_pool_fixup
+DEPTH:   Dis:2-, Geh:2-, Coc:2-, Tar:2-
+KMONS:   1 = hell sentinel
+KMONS:   2 = brimstone fiend
+KMONS:   3 = ice fiend
+KMONS:   4 = tzitzimitl
+KFEAT:   ! = iron_grate
+SHUFFLE: 1!/2l/3w/4m
+MAP
+    .....
+   .!!!!!.
+  ..!.1.!..
+ ...........
+.mm.......ll.
+.m.........l.
+.m4...{...2l.
+.m.........l.
+.mm.......ll.
+ ...........
+  ..w.3.w..
+   .wwwww.
+    .....
+ENDMAP
+
+NAME:  nicolae_hells_hurtin_island
+TAGS:  transparent no_pool_fixup
+DEPTH: Dis, !Dis:$, Geh, !Geh:$, Coc, !Coc:$, Tar, !Tar:$
+KMONS: 1> = tormentor / hellion / nothing
+KFEAT: ! = iron_grate
+KFEAT: > = >
+KMASK: 1> = opaque
+KMASK: 1> = no_monster_gen
+SUBST: l : lw!
+MAP
+.......
+.clllc.
+.l111l.
+.l1>1l.
+.l111l.
+.clllc.
+.......
+ENDMAP
+
+# Liches getting together in Hell to share news, tips, and the latest gossip
+NAME:  nicolae_hells_leichenhalle
+TAGS:  transparent no_pool_fixup
+DEPTH: Dis, !Dis:$, Geh, !Geh:$, Coc, !Coc:$, Tar, !Tar:$
+KMONS: 1 = ancient lich / dread lich / nothing
+KFEAT: ! = iron_grate
+: if you.in_branch("Dis") then
+SUBST: D : !c, E : Gvb, F : G., c : v
+TILE:  G = dngn_statue_iron
+: set_feature_name("granite_statue", "iron statue")
+: elseif you.in_branch("Geh") then
+SUBST: D : xc., E : Glcx., F : ..Gl
+: elseif you.in_branch("Coc") then
+KFEAT: 1 = W
+SUBST: D : wc, E : xw, F : .cw
+: elseif you.in_branch("Tar") then
+SUBST: D : cE, EF = ..G
+TILE:  G = dngn_crumbled_column
+: set_feature_name("granite_statue", "broken pillar")
+: end
+MAP
+  ccccccccccc
+  c1D1D1D1D1c
+ccc.........ccc
+cF...E...E...Fc
+c}.E...E...E..+
+cF...E...E...Fc
+ccc.........ccc
+  c1D1D1D1D1c
+  ccccccccccc
+ENDMAP
+
+# It's possible that the seeping gasses will end up harming the monsters on the
+# level, if they don't have the right resistances, but it's probably not worth
+# it to spend too much time luring monsters into the cloud zone, especially
+# since they might fall through the shaft and collapse it before you can use it.
+NAME:  nicolae_hells_seeping_fissure
+TAGS:  transparent luniq allow_dup no_monster_gen
+DEPTH: Dis, !Dis:$, Geh, !Geh:$, Coc, !Coc:$, Tar, !Tar:$
+KFEAT: ^ = shaft trap
+{{
+  local cloudtypes = {"calcifying dust", "foul pestilence", "negative energy",
+    "acidic fog", "thunder", "mutagenic fog", "degeneration", "flame",
+    "freezing vapour"}
+  local hellcloud = util.random_from( cloudtypes )
+  lua_marker( '^', fog_machine{ cloud_type = hellcloud, pow_min = 5,
+    pow_max = 40, delay = 15, size = 1, walk_dist = 1, spread_rate = 5 } )
+}}
+MAP
+.....
+.....
+..^..
+.....
+.....
+ENDMAP
+
+# This one is a bit silly, up until you get murdered by eight giants.
+NAME:    nicolae_hells_slight_mixup
```

Comment:
The initial gimmick/intended effect was just "fire giants with freezing weapons and frost giants with flaming weapons", and then I decided to go All The Way with it and do all the combinations of giant/weapon/brand. 

I can just make it a couple fire and frost giants hanging out. You'd lose the comedic mismatch of giant and brand, although you might not notice since you're getting murdered by eight giants.

Reply:
Yeah I'd just make it the mixed flavour giants hanging out.